client/my-sites/theme/main.jsx

```diff
@@ -782,20 +784,24 @@ class ThemeSheet extends Component {
 		);
 	};
 
-	onThemeActive = () => {
+	onAtomicThemeActive = () => {
 		if ( ! this.state.isAtomicTransferCompleted ) {
 			this.setState( {
 				isAtomicTransferCompleted: true,
 			} );
 
-			const { isAtomic, siteSlug } = this.props;
+			const { isAtomic, siteSlug, themeId } = this.props;
 			if ( ! isAtomic ) {
 				const newSiteSlug = siteSlug.replace( /\b.wordpress.com/, '.wpcomstaging.com' );
-				return page( `/theme/makoney/${ newSiteSlug }` );
+				return page( `/theme/${ themeId }/${ newSiteSlug }` );
 			}
 		}
 	};
 
+	onAtomicThemeActiveFailure = ( message ) => {
+		return this.props.errorNotice( message );
```

Comment:
Do we need to `return` here? Isn't the call to `errorNotice()` effectively a `dispatch()` call under the covers?